### DISCUSSION TITLE
街区厅小游戏鸣潮的技术力居然如此之“高”

### DISCUSSION THREAD

**Discussion by fengye2005** [share121/inter-knot#2351] [其他]
鸣潮启动器首页变成动态的，知道怎么实现的吗？放了 450 张图片播放，属实逆天，我直接改
![Image_1723735744564](https://github.com/user-attachments/assets/fb98cbf5-5fe1-444f-80d9-21a609e03a64)

**Comment by fengye2005** [comment]
添加证据和第二张图
![Image_1723736849083](https://github.com/user-attachments/assets/a505a808-6a5d-49fc-b72b-34a51bdbf546)
![Image_1723736855059](https://github.com/user-attachments/assets/f0d851c7-7a8f-4ecf-8e69-c17edba35764)

**Comment by jianjue1** [comment]
这思路倒是能用，就是实在是有些丑陋<img src="https://bbs-static.miyoushe.com/static/2024/05/10/d04c63ae9420d5bff5ec11d8fa2296a0_8388446090598690850.png" alt="派对客 呕" width="75" >
